cypress/integration/click-to-enter-spec.js

```diff
@@ -1,6 +1,21 @@
+import moment from 'moment'
+
 describe("Click To Enter component", () => {
 
   beforeEach(() => {
+    const today = moment().format('YYYY-MM-DD');
+    cy.intercept('GET', `https://api.nasa.gov/planetary/apod?date=${today}&api_key=j9VLjGbdXCRXtf61nCle9dLGtNzWVnNqUM1BNV86`, {
+        statusCode: 201,
+        body: {
+          "date": today,
```

Comment:
Hard coding this in was a good call. I appreciate that it will work pass tests on future dates too.